src/CMakeLists.txt

```diff
@@ -160,10 +160,12 @@ TARGET_LINK_LIBRARIES(lmms
 )
 
 FOREACH(LIB ${LMMS_REQUIRED_LIBS})
-    GET_TARGET_PROPERTY(INCLUDE_DIRS ${LIB} INTERFACE_INCLUDE_DIRECTORIES)
-    if (INCLUDE_DIRS)
-        TARGET_INCLUDE_DIRECTORIES(lmmsobjs PRIVATE ${INCLUDE_DIRS})
-    ENDIF()
+	IF(TARGET ${LIB})
```

Comment:
I already did this in 0dbdafc1f82ea0f6cb14a07bad4caab8e49643fc (albeit with inconsistent indentation...), it just hasn't been merged back to `master` yet.

Reply:
Thanks.  I think I'll keep it here and we can deal with the minor merge conflict when rebasing.